src/main/java/edu/greenblitz/bigRodika/commands/shooter/pidshooter/ShootByDashboard.java

```diff
@@ -20,11 +20,10 @@ public ShootByDashboard(double target) {
     public void initialize() {
         shooter.getPIDController().setIAccum(0);
 
-        this.target = shooter.getNumber("testing_target", 100);
-        this.obj = new PIDObject(shooter.getNumber("p", 0.01),
-                shooter.getNumber("i", 0),
-                shooter.getNumber("d", 0),
-                shooter.getNumber("f", 0.6));
+        this.obj = new PIDObject(shooter.getNumber("P", 0.01),
+                shooter.getNumber("I", 0),
+                shooter.getNumber("D", 0),
+                shooter.getNumber("F", 0.6));
```

Comment:
As far as we tested, we did not succeed to read information from the SmartDashboard, might wanna just use the robotMap.